examples/nextjs/src/components/new-todo-list.tsx

```diff
@@ -0,0 +1,42 @@
+import { TodoListCreateDocument } from "graphql/schema";
+import { useMutation } from "urql";
+import { useMemo, useState } from "react";
+
+const NewTodoList = () => {
+  const context = useMemo(() => ({ additionalTypenames: ["TodoList"] }), []);
+  const [title, setTitle] = useState<string>("");
+
+  const [{ fetching }, createTodoList] = useMutation(TodoListCreateDocument);
+
+  return (
+    <form
+      className="h-fit rounded-lg border-2 border-dashed border-gray-200 dark:border-gray-800 p-3 space-y-3 min-w-[300px]"
+      onSubmit={(e) => {
+        e.preventDefault();
+        createTodoList({ title }, context);
+        setTitle("");
+      }}
+    >
+      <h2 className="text-gray-900 dark:text-gray-300 font-semibold text-xl font-bold">
```

Comment:
`font-semibold` OR `font-bold`
Install TW intellisense to avoid these issues - https://tailwindcss.com/docs/editor-setup

Reply:
Done